src/rp2040/hardware_structs/include/hardware/structs/rtc.h

```diff
@@ -8,22 +10,101 @@
 #define _HARDWARE_STRUCTS_RTC_H
 
 #include "hardware/address_mapped.h"
-#include "hardware/platform_defs.h"
 #include "hardware/regs/rtc.h"
 
+// reference to datasheet: https://datasheets.raspberrypi.org/rp2040/rp2040-datasheet.pdf#tab-registerlist_rtc
+
+// BITMASK [BITRANGE]: FIELDNAME (RESETVALUE): DESCRIPTION
+// The REG macro is intended to help make the register navigable in your IDE (for example, using the "Go to Definition" feature)
+// _REG_(x) will link to the corresponding register in hardware/regs/rtc.h.
+
 typedef struct {
+    _REG_(RTC_CLKDIV_M1_OFFSET)
+    // Divider minus 1 for the 1 second counter
+    // 0x0000ffff [0-15]  : CLKDIV_M1 (0)
     io_rw_32 clkdiv_m1;
+
+    _REG_(RTC_SETUP_0_OFFSET)
+    // RTC setup register 0
+    // 0x00fff000 [12-23] : YEAR (0): Year
+    // 0x00000f00 [8-11]  : MONTH (0): Month (1
```

Comment:
Any idea why this line has been truncated so prematurely, rather than saying `Month (1..12)` ?

Reply:
i bet it is something to do with elipsis

Reply:
Created an issue for this https://github.com/raspberrypi/svd2struct/issues/2